src/Apps/W1/EDocumentConnectors/ForNAV/Test/app.json

```diff
@@ -0,0 +1,75 @@
+{
+	"id": "c848de8f-5f8b-4adf-8a7a-8de832b204dd",
+	"name": "E-Document Connector - FORNAV Tests",
+	"publisher": "Microsoft",
+	"brief": "Test of ForNAV Peppol",
+	"description": "Test of ForNAV Peppol",
+	"version": "1.0.0.0",
+	"privacyStatement": "https://go.microsoft.com/fwlink/?LinkId=724009",
+	"EULA": "https://go.microsoft.com/fwlink/?linkid=2009120",
+	"help": "https://go.microsoft.com/fwlink/?linkid=2204541",
+	"url": "https://go.microsoft.com/fwlink/?LinkId=724011",
+	"logo": "ExtensionLogo.png",
+	"contextSensitiveHelpUrl": "https://go.microsoft.com/fwlink/?linkid=2206603",
+	"dependencies": [
+		{
+			"id": "e1d97edc-c239-46b4-8d84-6368bdf67c8b",
+			"name": "E-Document Core",
+			"publisher": "Microsoft",
+			"version": "26.0.0.0"
+		},
+		{
+			"id": "e1d97edc-c239-46b4-8d84-6368bdf67c8c",
+			"name": "E-Document Core Tests",
+			"publisher": "Microsoft",
+			"version": "26.0.0.0"
+		},
+		{
+			"id": "5d86850b-0d76-4eca-bd7b-951ad998e997",
+			"name": "Tests-TestLibraries",
+			"publisher": "Microsoft",
+			"version": "26.0.0.0"
+		},
+		{
+			"id": "9856ae4f-d1a7-46ef-89bb-6ef056398228",
+			"name": "System Application Test Library",
+			"publisher": "Microsoft",
+			"version": "26.0.0.0"
+		},
+		{
+			"id": "5095f467-0a01-4b99-99d1-9ff1237d286f",
+			"publisher": "Microsoft",
+			"name": "Library Variable Storage",
+			"version": "26.0.0.0"
+		},
+		{
+			"id": "40860557-a18d-42ad-aecb-22b7dd80dc80",
+			"name": "Permissions Mock",
+			"publisher": "Microsoft",
+			"version": "26.0.0.0"
+		},
+		{
+			"id": "0addc017-80c4-40ad-bac6-5852f4fc4c55",
+			"name": "E-Document Connector - FORNAV",
+			"publisher": "Microsoft",
+			"version": "1.0.0.0"
+		}
+	],
+	"screenshots": [],
+	"platform": "26.0.0.0",
+	"application": "26.0.0.0",
+	"idRanges": [
+		{
+			"from": 6427,
+			"to": 6430
+		}
+	],
+	"runtime": "15.0",
```

Comment:
I don't think we usually define this in the app.json 🤔 I don't see any other apps in BCApps with this property. Is it needed?